service/src/test/java/bio/terra/workspace/pact/landingzones/LzReadApiTest.java

```diff
@@ -0,0 +1,321 @@
+        .toPact();
+  }
+
+  @Pact(consumer = "workspacemanager", provider = "terra-landing-zone-service")
+  public RequestResponsePact listLandingZoneDefinitions(PactDslWithProvider builder) {
+    var definitionsResponseShape =
+        new PactDslJsonBody()
+            .eachLike("landingzones")
+            .stringType("definition")
+            .stringType("name")
+            .stringType("description")
+            .stringType("version");
+
+    return builder
+        .uponReceiving("A request to list landing zone defs")
+        .method("GET")
+        .path("/api/landingzones/definitions/v1/azure")
+        .willRespondWith()
+        .body(definitionsResponseShape)
+        .status(HttpStatus.OK.value())
+        .toPact();
+  }
+
+  @Pact(consumer = "workspacemanager", provider = "terra-landing-zone-service")
+  public RequestResponsePact listAzureLandingZoneResources(PactDslWithProvider builder) {
+    var resourcesResponseShape =
+        new PactDslJsonBody().uuid("id").eachLike("resources", resourcesByTypeShape);
+
+    return builder
+        .given("An existing landing zone")
+        .uponReceiving("A request to list the landing zone resources")
+        .method("GET")
+        .pathFromProviderState(
+            "/api/landingzones/v1/azure/${LANDING_ZONE_ID}/resources",
+            "/api/landingzones/v1/azure/%s/resources".formatted(LANDING_ZONE_ID))
+        .willRespondWith()
+        .body(resourcesResponseShape)
+        .status(HttpStatus.OK.value())
+        .toPact();
+  }
+
+  @Pact(consumer = "workspacemanager", provider = "terra-landing-zone-service")
+  public RequestResponsePact getResourceQuota(PactDslWithProvider builder) {
+    var quotaResponseShape =
+        new PactDslJsonBody()
+            .uuid("landingZoneId")
+            .stringType("azureResourceId")
+            .stringType("resourceType")
+            .object("quotaValues");
+
+    return builder
+        .given("An existing landing zone")
+        .uponReceiving("A request to get a resource's quota")
+        .method("GET")
+        .pathFromProviderState(
+            "/api/landingzones/v1/azure/${LANDING_ZONE_ID}/resource-quota",
+            "/api/landingzones/v1/azure/%s/resource-quota".formatted(LANDING_ZONE_ID))
+        .query(String.format("azureResourceId=%s", azureResourceId))
+        .willRespondWith()
+        .body(quotaResponseShape)
+        .status(HttpStatus.OK.value())
+        .toPact();
+  }
+
+  @Test
+  @PactTestFor(pactMethod = "listLandingZonesForBillingProfile", pactVersion = PactSpecVersion.V3)
+  void listLandingZonesForBillingProfile() throws InterruptedException {
+    var result =
+        landingZoneService.listLandingZonesByBillingProfile(
+            new BearerToken("fake"), BILLING_PROFILE_ID);
+
+    assertNotNull(result);
+    assertThat(result.getLandingzones(), is(not(empty())));
+  }
+
+  @Test
+  @PactTestFor(pactMethod = "listLandingZones", pactVersion = PactSpecVersion.V3)
```

Comment:
One thing that some people have found to improve readability, is to locate the `@Pact` method next to the corresponding test(s).

I don't think it's necessary or anything like that, just something you might prefer.